inference-engine/src/vpu/graph_transformer/src/stages/interpolate.cpp

```diff
@@ -51,12 +51,31 @@ void FrontEnd::parseInterpolate(const Model& model, const ie::CNNLayerPtr& _laye
             if (cmp(interpolateMode, "nearest")) {
                 // current "Resample" supports the following "Interpolate" modes only:
                 // coordinate_transformation_mode = half_pixel; nearest_mode = round_prefer_ceil;
+                // coordinate_transformation_mode = asymmetric; nearest_mode = floor;
                 // other "Interpolate" modes are translated to the default ones
                 const auto antialias = _layer->GetParamAsBool("antialias", false);
+                const auto coordinate_transformation_mode = _layer->GetParamAsString("coordinate_transformation_mode", "half_pixel");
+                const auto nearest_mode = _layer->GetParamAsString("nearest_mode", "round_prefer_floor");
+                InterpolateCoordTransMode coordinateTransformationMode = InterpolateCoordTransMode::half_pixel;
+                InterpolateNearestMode nearestMode = InterpolateNearestMode::round_prefer_ceil;
+
+                if (cmp(coordinate_transformation_mode, "asymmetric")) {
+                    coordinateTransformationMode = InterpolateCoordTransMode::asymmetric;
```

Comment:
Are there any other `coordinateTransformationMode`'s or `nearestMode`'s? If yes, I think we should also throw on unsupported ones.

Reply:
No, we should do not throw an exception. We should use default mode.

Reply:
Ok, got it.